mongo/doc.go

```diff
@@ -105,8 +105,8 @@
 //
 // Note: Auto encryption is an enterprise-only feature.
 //
-// The libmongocrypt C library is required when using client-side encryption. libmongocrypt version 1.3.0 or higher is
-// required when using driver version 1.8.0 or higher. To install libmongocrypt, follow the instructions for your
+// The libmongocrypt C library is required when using client-side encryption. libmongocrypt version 1.5.0 or higher is
+// required when using driver version 1.10.0 or higher. To install libmongocrypt, follow the instructions for your
```

Comment:
Does it make sense to retain information about required libmongocrypt versions for previous driver versions?

E.g.
```go
// The libmongocrypt C library is required when using client-side encryption. Specific versions of libmongocrypt
// are required for different versions of the Go Driver:
// - Go Driver v1.8.0 requires libmongocrypt v1.3.0 or higher
// - Go Driver v1.10.0 requires libmongocrypt v1.5.0 or higher
```
It's easier to make sense of the information when it's all in one place, but the docs for v1.10.0 are technically only relevant for people using v1.10.0, so the information about previous versions isn't technically necessary.